tests/integration_tests/build/test_unittests.py

```diff
@@ -17,6 +17,7 @@
     "target",
     TARGETS
 )
+@pytest.mark.concurrency(1)
```

Comment:
why do we add this restriction? Do we need to also add it for `test_coverage.py`?

Reply:
Some unit tests pass criteria are based on time measurement.